test/e2e/e2e_test.go

```diff
@@ -126,9 +126,27 @@ var _ = ginkgo.Describe("E2E Test Suite", func() {
 		testsuites.InitGcsFuseCSIGCSFuseIntegrationFileCacheParallelDownloadsTestSuite,
 	}
 
-	testDriverHNS := specs.InitGCSFuseCSITestDriver(c, m, *bucketLocation, *skipGcpSaTest, true)
+	testDriverHNS := specs.InitGCSFuseCSITestDriver(c, m, *bucketLocation, *skipGcpSaTest, true, false)
 
 	ginkgo.Context(fmt.Sprintf("[Driver: %s HNS]", testDriverHNS.GetDriverInfo().Name), func() {
 		storageframework.DefineTestSuites(testDriverHNS, GCSFuseCSITestSuitesHNS)
 	})
+
+	testDriverHNSAndGrpc := specs.InitGCSFuseCSITestDriver(c, m, *bucketLocation, *skipGcpSaTest, true, true)
+
+	ginkgo.Context(fmt.Sprintf("[Driver: %s HNS + GRPC]", testDriverHNSAndGrpc.GetDriverInfo().Name), func() {
+		storageframework.DefineTestSuites(testDriverHNSAndGrpc, GCSFuseCSITestSuitesHNS)
+	})
+
+	GCSFuseCSITestSuitesGRPC := []func() storageframework.TestSuite{
```

Comment:
@saikat-royc any other testsuites we should consider running with grpc? Some of my considerations could be istio and metric tests.

Reply:
As discussed let's create a new prow job spec with gRPC enabled driver, and run the existing test suites. So we can keep the timeout of the current prow job to its existing value of 4h, and get high coverage of the existing testsuite test cases with gRPC